mage_integrations/mage_integrations/destinations/snowflake/__init__.py

```diff
@@ -21,8 +21,8 @@
 
 def convert_array(value, column_settings):
     if type(value) is list and value:
-        value_string = ', '.join([str(i) for i in value])
-        return value_string
+        value_string = ', '.join([f"'{str(i)}'" for i in value])
```

Comment:
Could the value be numbers?

Reply:
@wangxiaoyou1993 it can be any value, but it doesn't seem like we really check for that

Reply:
and it seems like we're already trying to convert everything to a string